### ISSUE TITLE
Brand new install of PowerShell 7.2.0, PSReadline failure immediately after opening new window

### ISSUE THREAD

**Issue by johnmccrae** [PowerShell/PSReadLine#2986]
### Prerequisites

- [X] Write a descriptive title.
- [X] Make sure you are able to repro it on the [latest released version](https://www.powershellgallery.com/packages/PSReadLine)
- [X] Search the existing issues, especially the pinned issues.

### Exception report

```console
[Admin] ¥ DESKTOP-79SOATF {C:\Users\JohnMcCrae}
Oops, something went wrong.
Please report this bug with ALL the details below, including both the 'Environment' and 'Exception' sections.
Please report on GitHub: https://github.com/PowerShell/PSReadLine/issues/new?template=Bug_Report.md
Thank you!

### Environment
PSReadLine: 2.2.0-beta1
PowerShell: 7.2.0
OS: Microsoft Windows 10.0.22000
BufferWidth: 271
BufferHeight: 83

Last 0 Keys


```

### Exception
```
System.TypeLoadException: Could not load type 'System.Management.Automation.Subsystem.PredictionResult' from assembly 'Microsoft.PowerShell.PSReadLine.Polyfiller, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.
   at Microsoft.PowerShell.PSConsoleReadLine.PredictionViewBase.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.PredictionInlineView.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.Prediction.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.Initialize(Runspace runspace, EngineIntrinsics engineIntrinsics)
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics, CancellationToken cancellationToken)
```
```


### Screenshot


![Screenshot 2021-11-09 102403](https://user-images.githubusercontent.com/4855887/140982539-9534e740-2f22-4ba2-9a97-2e59c962e326.png)



### Environment data

```console
PSReadLine: 2.2.0-beta1
PowerShell: 7.2.0
OS: Microsoft Windows 10.0.22000
BufferWidth: 271
BufferHeight: 83

*** NOTE - the OS Version above is wrong. I am running Windows 11. Both Windows PowerShell and PowerShell report Windows 10. Weird.

after I updated PSReadline (see article below), I was able to run the script above and got this instead.

PS Version: 7.2.0
PS HostName: ConsoleHost (Windows Terminal)
PSReadLine Version: 2.1.0
PSReadLine EditMode: Windows
OS: 10.0.22000.1 (WinBuild.160101.0800)
BufferWidth: 271
BufferHeight: 83
```

### Steps to reproduce

1. Install PowerShell 7.2.0
2. Open a new PowerShell window

### Expected behavior

No exceptions from PSReadline, be able to actually use PowerShell, 

### Actual behavior

Exceptions from PSReadline

Also I cannot run PowerShell now. Every new instance dies immediately with the error above. My $profile is pretty beefy and calls PSReadline for the keyboard history. I'll need to uninstall this beta version and fall back. 

This is really borked now. `pwsh.exe -noprofile` is not working either so I can't get to a clean prompt and try something. 

Correction - I found this article and it sorted me out. [Go Here to update PSReadline](https://www.hanselman.com/blog/powershell-720-could-not-load-type-systemmanagementautomationsubsystempredictionresult)

**Comment by martins-vds** [comment]
It happened to me as well. Duplicate of #2985

**Comment by andyleejordan** [comment]
Hi, I see you're using an old version of PSReadLine. Please upgrade to use the latest [2.2.0-beta4 version of PSReadLine](https://www.powershellgallery.com/packages/PSReadLine/2.2.0-beta4) and let us know if this continues to reproduce. Thanks!

**Comment by daxian-dbw** [comment]
This issue was fixed in 2.2.0-beta3 version of PSReadLine. You can fix this by upgrading to the latest [2.2.0-beta4 version of PSReadLine](https://www.powershellgallery.com/packages/PSReadLine/2.2.0-beta4). Instructions for doing so:
1. stop all instances of `pwsh`.
2. from `cmd.exe` run: `pwsh -noprofile -command "Install-Module PSReadLine -AllowPrerelease -Force"`

--------

If you want to remove that beta version of PSReadLine and use the 2.1.0 version of PSReadLine that's shipped with PowerShell 7.2, you can:
1. run `pwsh -noprofile -noninteractive` to start `pwsh` without loading PSReadLine
2. run `Uninstall-Module -Name PSReadLine -RequiredVersion <2.2.0-beta1 or 2.2.0-beta2> -AllowPrerelease` to remove the module. Or, you can manually remove that module folder.
